arch/arm/boot/dts/overlays/revpi-core-s-2022-overlay.dts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Device tree overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Core S (2022)
+ */
+
+ #include "revpi-core-2022-overlay.dts"
+
+/{
+	compatible = "brcm,bcm2711";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-core-s", "kunbus,revpi-core",
```

Comment:
we have kunbus,revpi-core-se-2022, and kunbus,revpi-core-2022
but here it is kunbus,revpi-core-s ?